diff --git a/scripts/16.4-react-core-old.js b/scripts/16.4-react-core.js
index 4554cb1..7f3376b 100644
--- a/scripts/16.4-react-core-old.js
+++ b/scripts/16.4-react-core.js
@@ -203,7 +203,7 @@
                         e = e ? e[t[n]] : a;
                     return e === a ? r : e;
                 },
-                d = function e(t) {
+                d = function e_(t) {
                     var r = {},
                         n = function (e) {
                             var n = {},
@@ -258,7 +258,7 @@
                         u.length > 1 &&
                             u.forEach(function (r) {
                                 var a;
-                                n[r] = e((((a = {})[r] = t[r]), a));
+                                n[r] = e_((((a = {})[r] = t[r]), a));
                             }),
                         n
                     );
@@ -967,12 +967,12 @@
                 );
             }
             var x = (function () {
-                    function e(e) {
+                    function e_(e) {
                         (this.groupSizes = new Uint32Array(512)),
                             (this.length = 512),
                             (this.tag = e);
                     }
-                    var t = e.prototype;
+                    var t = e_.prototype;
                     return (
                         (t.indexOfGroup = function (e) {
                             for (var t = 0, r = 0; r < e; r++)
@@ -1112,7 +1112,7 @@
                     );
                 },
                 D = (function () {
-                    function e(e) {
+                    function e_(e) {
                         var t = (this.element = $(e));
                         t.appendChild(document.createTextNode("")),
                             (this.sheet = (function (e) {
@@ -1131,12 +1131,12 @@
                             })(t)),
                             (this.length = 0);
                     }
-                    var t = e.prototype;
+                    var t = e_.prototype;
                     return (
-                        (t.insertRule = function (e, t) {
+                        (t.insertRule = function (e__, t) {
                             try {
                                 return (
-                                    this.sheet.insertRule(t, e),
+                                    this.sheet.insertRule(t, e__),
                                     this.length++,
                                     !0
                                 );
@@ -1153,15 +1153,15 @@
                                 ? t.cssText
                                 : "";
                         }),
-                        e
+                        e_
                     );
                 })(),
                 I = (function () {
-                    function e(e) {
+                    function e_(e) {
                         var t = (this.element = $(e));
                         (this.nodes = t.childNodes), (this.length = 0);
                     }
-                    var t = e.prototype;
+                    var t = e_.prototype;
                     return (
                         (t.insertRule = function (e, t) {
                             if (e <= this.length && e >= 0) {
@@ -1184,14 +1184,14 @@
                                 ? this.nodes[e].textContent
                                 : "";
                         }),
-                        e
+                        e_
                     );
                 })(),
                 M = (function () {
-                    function e(e) {
+                    function e_(e) {
                         (this.rules = []), (this.length = 0);
                     }
-                    var t = e.prototype;
+                    var t = e_.prototype;
                     return (
                         (t.insertRule = function (e, t) {
                             return (
@@ -1205,13 +1205,13 @@
                         (t.getRule = function (e) {
                             return e < this.length ? this.rules[e] : "";
                         }),
-                        e
+                        e_
                     );
                 })(),
                 z = S,
                 W = { isServer: !S, useCSSOMInjection: !0 },
                 U = (function () {
-                    function e(e, t, r) {
+                    function e_(e, t, r) {
                         void 0 === e && (e = y),
                             void 0 === t && (t = {}),
                             (this.options = h({}, W, {}, e)),
@@ -1239,15 +1239,15 @@
                                     }
                                 })(this));
                     }
-                    e.registerId = function (e) {
+                    e_.registerId = function (e) {
                         return j(e);
                     };
-                    var t = e.prototype;
+                    var t = e_.prototype;
                     return (
                         (t.reconstructWithOptions = function (t, r) {
                             return (
                                 void 0 === r && (r = !0),
-                                new e(
+                                new e_(
                                     h({}, this.options, {}, t),
                                     this.gs,
                                     (r && this.names) || void 0
@@ -1339,7 +1339,7 @@
                                 return n;
                             })(this);
                         }),
-                        e
+                        e_
                     );
                 })(),
                 H = /(a)(d)/gi,
@@ -1370,7 +1370,7 @@
             }
             var V = J("5.3.11"),
                 X = (function () {
-                    function e(e, t, r) {
+                    function e_(e, t, r) {
                         (this.rules = e),
                             (this.staticRulesId = ""),
                             (this.isStatic =
@@ -1381,7 +1381,7 @@
                             U.registerId(t);
                     }
                     return (
-                        (e.prototype.generateAndInjectStyles = function (
+                        (e_.prototype.generateAndInjectStyles = function (
                             e,
                             t,
                             r
@@ -1443,7 +1443,7 @@
                             }
                             return a.join(" ");
                         }),
-                        e
+                        e_
                     );
                 })(),
                 G = /^\s*\/\/.*$/gm,
@@ -1595,7 +1595,7 @@
                 );
             }
             var ei = (function () {
-                    function e(e, t) {
+                    function e_(e, t) {
                         var r = this;
                         (this.inject = function (e, t) {
                             void 0 === t && (t = er);
@@ -1615,10 +1615,10 @@
                             (this.rules = t);
                     }
                     return (
-                        (e.prototype.getName = function (e) {
+                        (e_.prototype.getName = function (e) {
                             return void 0 === e && (e = er), this.name + e.hash;
                         }),
-                        e
+                        e_
                     );
                 })(),
                 es = /([A-Z])/,
@@ -1655,7 +1655,7 @@
                         ? (e.inject(r, n), e.getName(n))
                         : e
                     : p(e)
-                    ? (function e(t, r) {
+                    ? (function e_(t, r) {
                           var n,
                               a = [];
                           for (var o in t)
@@ -1665,7 +1665,7 @@
                                   g(t[o])
                                       ? a.push(ed(o) + ":", t[o], ";")
                                       : p(t[o])
-                                      ? a.push.apply(a, e(t[o], o))
+                                      ? a.push.apply(a, e_(t[o], o))
                                       : a.push(
                                             ed(o) +
                                                 ": " +
@@ -1759,7 +1759,7 @@
             eR.Consumer;
             var eC = {},
                 ek = function (e) {
-                    return (function e(t, r, a) {
+                    return (function e_(t, r, a) {
                         if (
                             (void 0 === a && (a = y),
                             !(0, n.isValidElementType)(r))
@@ -1770,10 +1770,10 @@
                         };
                         return (
                             (o.withConfig = function (n) {
-                                return e(t, r, h({}, a, {}, n));
+                                return e_(t, r, h({}, a, {}, n));
                             }),
                             (o.attrs = function (n) {
-                                return e(
+                                return e_(
                                     t,
                                     r,
                                     h({}, a, {
@@ -1785,7 +1785,7 @@
                             }),
                             o
                         );
-                    })(function e(t, r, n) {
+                    })(function e_(t, r, n) {
                         var o = b(t),
                             i = !eS(t),
                             s = r.attrs,
@@ -1961,7 +1961,7 @@
                                         return a;
                                     })(r, ["componentId"]),
                                     i = a && a + "-" + (eS(t) ? t : eb(v(t)));
-                                return e(
+                                return e_(
                                     t,
                                     h({}, o, { attrs: E, componentId: i }),
                                     n
@@ -1971,7 +1971,7 @@
                                 get: function () {
                                     return this._foldedDefaultProps;
                                 },
-                                set: function (e) {
+                                set: function (e_) {
                                     this._foldedDefaultProps = o
                                         ? (function e(t) {
                                               for (
@@ -2011,7 +2011,7 @@
                                               }
                                               return t;
                                           })({}, t.defaultProps, e)
-                                        : e;
+                                        : e_;
                                 },
                             }),
                             Object.defineProperty(j, "toString", {
@@ -2175,13 +2175,13 @@
                 ek[e] = ek(e);
             });
             var eN = (function () {
-                function e(e, t) {
+                function e_(e, t) {
                     (this.rules = e),
                         (this.componentId = t),
                         (this.isStatic = Y(e)),
                         U.registerId(this.componentId + 1);
                 }
-                var t = e.prototype;
+                var t = e_.prototype;
                 return (
                     (t.createStyles = function (e, t, r, n) {
                         var a = n(ef(this.rules, t, r, n).join(""), ""),
@@ -2196,7 +2196,7 @@
                             this.removeStyles(e, r),
                             this.createStyles(e, t, r, n);
                     }),
-                    e
+                    e_
                 );
             })();
             function ej(e) {
@@ -5391,7 +5391,7 @@
                 return {
                     path: e,
                     Component: t,
-                    coreLoader: async function e({ location: e }) {
+                    coreLoader: async function e_({ location: e }) {
                         let t;
                         try {
                             let r = `${e.pathname}${e.search}`;
